glean-core/python/glean/net/ping_upload_worker.py

```diff
@@ -111,7 +111,7 @@ def _process(data_dir: Path, application_id: str, configuration) -> bool:
         cfg = _ffi.make_config(
             data_dir,
             application_id,
-            True,
+            False,
```

Comment:
Can we document why and what we're passing here?
Should call out that we're disabling upload because the process should not write data and that this does _not_ reset the client id